__posts/2022-06-02-release-2.2.0.md

```diff
@@ -30,15 +30,15 @@ For more help with installation see the [installation instructions in the User G
 
 ## Notable new additions
 
- - The `frames` argument was added to AnalysisBase-derived classes (i.e. modern analysis classes) allowing for specific frames to be defined when running an analysis. (PR #3415)
+ - The `frames` argument was added to AnalysisBase-derived classes (i.e. modern analysis classes) allowing for specific frames to be defined when running an analysis. This new feature allows one to run per-frame analysis on arbitrary subsets of trajectory frames. (PR #3415)
  - Python wheels are now made available through PyPI for x86_64 architectures (Issue #1300, PR #3680)
```

Comment:
We should also highlight Tyler's work on getting M1 builds to work (and thank Mike for testing it on release).

Reply:
added